crates/libduckdb-sys/src/lib.rs

```diff
@@ -14,6 +14,24 @@ pub use bindings::*;
 pub const DuckDBError: duckdb_state = duckdb_state_DuckDBError;
 pub const DuckDBSuccess: duckdb_state = duckdb_state_DuckDBSuccess;
 
+use std::slice;
+impl From<&duckdb_string_t> for String {
```

Comment:
`DuckDbString` from https://github.com/duckdb/duckdb-rs/pull/361 might be beneficial here, instead of preemptively converting to Rust String